README.md

```diff
@@ -95,6 +102,11 @@ Client library will be placed in:
 lib/libpulsar.so
 lib/libpulsar.a
 ```
+Examples can be run on:
+
+```
+examples/
```

Comment:
These words can be improved. Either `Examples are located at ...` or using some copy-paste-able example running commands.